02-peer-protocol.md

```diff
@@ -562,10 +574,14 @@ The receiving node:
   - if `fee_satoshis` is equal to its previously sent `fee_satoshis`:
     - SHOULD sign and broadcast the final closing transaction.
     - MAY close the connection.
-  - otherwise, if `fee_satoshis` is greater than
-the base fee of the final commitment transaction as calculated in
-[BOLT #3](03-transactions.md#fee-calculation):
-    - MUST fail the connection.
+  - if `option_anchor_outputs` applies to the final commitment transaction:
+    - if `fee_satoshis` is less than the base fee of the final commitment transaction
+      - MUST fail the connection.
+  - otherwise (no `option_anchor_outputs`):
```

Comment:
> otherwise _if_ no .. 

?

Reply:
Word `if` added. Assuming that was what question mark was about.